### ISSUE TITLE
Mihon not tracking Komga read

### ISSUE THREAD

**Issue by ascheel319** [mihonapp/mihon#55]
### Steps to reproduce

1. Install Komga extension and sign in.
2. Turn enhanced tracking on.
3. Track a series.
4. Read a chapter in said series.
5. Check Komga webUI.

### Expected behavior

It should mark the chapter that was just read as read in Komga through an API call.

### Actual behavior

The chapter is not marked as read.

### Crash logs

App does not crash

### Mihon version

0.16.0

### Android version

Android 14

### Device

Google Pixel 7 Pro & Kindle Fire HD 8

### Other details

I was talking in the Tachiyomi discord before everything happened and the people there said all my settings looked fine. I went over to the Komga server and they said everything looked fine. The API call works on Komga's end so it's most likely how Tachiyomi (now Mihon) does the call. To remove my server and reverse proxy from the equation, I added the demo server and read something from there. I looked at the webpage version and it did not get marked as read. I refreshed and reloaded everything and nothing got it to update. I believe that while the content is from an official extension, tracking is handled by the app itself so that is why I am submitting an issue here instead of with the Komga extension.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.0](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Ore-an** [comment]
I had a similar but different problem, komga didn't appear under advanced trackers when I installed the extension from one of the cloned extension repos, which I guess has to do with how Tachiyomi/Mihon decides which extension is komga (maybe it's this line?):

https://github.com/mihonapp/mihon/blob/0a2df21c5b5a16a63cd80ff3cd256eb0521450fb/app/src/main/java/eu/kanade/tachiyomi/data/track/komga/Komga.kt#L96

I solved it by downloading the extension apk directly from https://tachiyomi.org/extensions/ (which appears simply as all.komga), maybe that could work for you too? If you install it in parallel with the extension you have now it will pick up the config from the existing one, if it works you can just remove the previous one.

**Comment by ascheel319** [comment]
I downloaded it from the official repo to begin with. It shows up in my enhanced trackers but doesn't tell Komga when something was read.

**Comment by mrjfilippo** [comment]
I have the same issue. Komga shows up in Enhanced Trackers, but progress doesn't sync.

**Comment by Gabigeek** [comment]
Same issue, everything up to date. When I try to enable tracking, I get the "No Match Found" error :
![Screenshot_20240723_034401_Mihon](https://github.com/user-attachments/assets/2609b7b7-dee2-40c5-a2e9-2e1bbb867f4b)

**Comment by AntsyLich** [comment]
This should be fixed in latest version. Make a new issue if it isn't
